b3lb/rest/task/recording.py

```diff
@@ -79,18 +81,21 @@ def render_record(record_set: RecordSet):
             profile_relations = SecretRecordProfileRelation.objects.filter(secret=record_set.secret)
             if profile_relations.count() > 0:
                 for profile_relation in profile_relations:
-                    render_by_profile(record_set, profile_relation.record_profile, tempdir)
+                    record_id = render_by_profile(record_set, profile_relation.record_profile, tempdir)
             else:
                 for record_profile in RecordProfile.objects.filter(is_default=True):
-                    render_by_profile(record_set, record_profile, tempdir)
+                    record_id = render_by_profile(record_set, record_profile, tempdir)
+
+    # Greenlight v3 sends a record ready URL, Greenlight v2 doesn't send one
+    if record_set.recording_ready_origin_url:
+        body = {"signed_parameters": jwt_encode({"meeting_id": record_set.meta_meeting_id, "record_id": record_id}, record_set.secret.get_valid_secret)}
+        response = post(record_set.recording_ready_origin_url, json=body)
+        print(f"Send record ready callback to: {record_set.recording_ready_origin_url} with return code: {response.status_code}")
```

Comment:
Looks like orphan debugging print?

Reply:
Looks like but it's just a logging print